### DISCUSSION TITLE
Importing a DXF with opend Faces as wire Model

### DISCUSSION THREAD

**Discussion by DeBalog** [CadQuery/cadquery#1735] [Q&A]
I created a DXF-Model made of lines, which I would like to import in cadquery as a peace of art:
![grafik](https://github.com/user-attachments/assets/12756d31-78da-45c0-8bba-6261b6423645)

While Trying to import this with the following code:

``` python
import cadquery as cq

result = cq.importers.importDXF("dish_test.dxf").wires().toPending().extrude(2)
show_object (result)
```

I'm getting the Error: "Cannot build face(s): outer wire is not closed".

How can I import the DXF as it is?

**Comment by jmwright** [comment]
There are many open and overlapping wires. I don't think the CAD kernel is ever going to be able to handle that. I think you'll need to simplify the DXF in sections, extrude those separately, and then union all the separate parts together. Extrude wants a clean, closed outer profile, so try for that.

**Comment by DeBalog** [comment]
I think I won't go this way.
